lang/interpret_test/TestAstFunc2/class-include-as-class2.txtar

```diff
@@ -0,0 +1,17 @@
+-- main.mcl --
+class c1($b) {
+	if $b {	# scope doesn't leak up and out of `if` statement!
+		class inner() {
+			test "t1" {}
+		}
+	} else {
+		class inner() {
+			test "t2" {}
+		}
+	}
+}
+
+include c1 as i1
+include i1.inner
+-- OUTPUT --
+# err: errSetScope: class `c1` expected 1 args but got 0
```

Comment:
Neat that it can find the fix, but too bad that it doesn't realize it's a test which is making sure that "bug" shows up as an error.

Reply:
@coderabbitai Can you tell me anything useful or interesting about the resources in the mcl code? This is a test.

Reply:
I asked about the _resources_ not the statements.

Reply:
To be fair, if I wasn't already familiar with the project, I wouldn't know which lines are resources either.

Reply:
@gelisam Yup, I was testing it to see if it was doing anything beyond simple pattern recognition-level stuff =D
I also get the feeling a lot of analysis is based on text docs and comments, which worries me because when do the docs or comments ever match the code :sweat_smile: